Dockerfile

```diff
@@ -20,3 +22,7 @@ RUN gem install rake --version =12.3.0 \
     && gem install kitchen-terraform --version 3.0.0 \
     && gem install test-kitchen --version 1.16.0
 WORKDIR /tf-test/module
+
+RUN wget https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-amd64.tar.gz >/dev/null 2>&1
```

Comment:
Can we follow same download and unzip pattern as terraform above? Don't need to have 2 different coding style for the similar thing on the same file.

Reply:
Will try curl.